analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -140,6 +140,36 @@ class MavenSupportTest : WordSpec({
                 path = "boringssl-static"
             )
         }
+
+        "handle GitHub URLs with double 'git:' prefix" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git:git:github.com/MarkusAmshove/Kluent.git"
```

Comment:
pick 4aca038e8 refactor: Always get projects and packages via OrtResult getters

Reply:
> pick 4aca038e8 refactor: Always get projects and packages via OrtResult getters

Hmm, I'm not sure what you're trying to tell me 👀

Reply:
LOL, this was a copy and paste mistake.